sormas-backend/src/main/java/de/symeda/sormas/backend/labmessage/LabMessageService.java

```diff
@@ -1,5 +1,9 @@
 package de.symeda.sormas.backend.labmessage;
 
+import java.util.Calendar;
+import java.util.GregorianCalendar;
```

Comment:
Out of curiosity: Why did you directly import and use the gregorian calendar here, while relying on the Abstract Calendar in the LabMessageIndexDto?

Reply:
Adjusted to use `Calendar.getInstance`, too